adapters/Pangle/PangleAdapter/Bidding/GADPangleRTBBannerRenderer.m

```diff
@@ -21,24 +21,21 @@
 
 @interface GADPangleRTBBannerRenderer () <PAGBannerAdDelegate>
 
+@property (nonatomic, copy) GADMediationBannerLoadCompletionHandler loadCompletionHandler;
+@property (nonatomic, strong) PAGBannerAd *bannerAd;
+@property (nonatomic, weak) id<GADMediationBannerAdEventDelegate> delegate;
+
```

Comment:
@wangyuanyuanyy What is the exact error that you saw that led to this change?

Reply:
![img_v2_e82ca309-01bb-4468-9154-911f6ca6dbeg](https://user-images.githubusercontent.com/98148726/191679989-ec3c5bfa-75df-46b2-a601-58b6b1e75c47.jpg)
 If for some reason “ self ” is released earlier，`self.xxx `results in determining whether the method will be called, but `self->xxx` will be called directly, resulting in a crash.